test/labeling/model/test_label_model.py

```diff
@@ -218,6 +218,59 @@ def test_predict_proba(self):
         true_probs = np.array([[0.99, 0.01], [0.99, 0.01]])
         np.testing.assert_array_almost_equal(probs, true_probs)
 
+    def test_predict(self):
+        L = np.array([[1, 2, 1], [1, 2, 1]])
+        label_model = self._set_up_model(L)
+
+        label_model.mu = nn.Parameter(label_model.mu_init.clone())
+        preds = label_model.predict(csr_matrix(L))
+
+        true_preds = np.array([1, 1])
+        np.testing.assert_array_equal(preds, true_preds)
+
+    def test_break_ties(self):
+        # abtains with ties
+        label_model = LabelModel(k=3)
+        probs = np.array([[0.33, 0.33, 0.33]])
+        preds = label_model._break_ties(probs, break_ties="abstain")
+        true_preds = np.array([0.0])
+        np.testing.assert_array_equal(preds, true_preds)
+
+        # random with ties
+        probs = np.array([[0.33, 0.33, 0.33]])
+        random_preds = []
+        for seed in range(10):
```

Comment:
What invariant of LabelModel are we trying to test here?

Reply:
whether the random tie breaking is truly random